### ISSUE TITLE
Document `show` command

### ISSUE THREAD

**Issue by AlexVonB** [MalteT/2a-emulator#5]
Hi Malte,

first things first, thank you for your emulator!

While using it, I could not find documentation on the `show` command. Only after I searched the code and found https://github.com/MalteT/2a-emulator/blob/946d5cedb472f85d2dea410b28bb038999bf2009/src/tui/input/parser.rs#L267-L268 it all made sense to me. I would have added a section to the Benutzerhandbuch.pdf, but I could not find the LaTeX sources.

My proposed changes would include one or both of the following:

- Add a section covering the commands into Readme or Benutzerhandbuch.
- Show possible parameters on the right hand side when typing `show `, just like when typing `set `: https://github.com/MalteT/2a-emulator/blob/5f57cd8fff7e08d468b190638e377ad5f54b9680/src/tui/interface.rs#L334

Let me know, if I can help you!

Best
Alex

**Comment by MalteT** [comment]
Hey, @AlexVonB, 

thanks for the issue! I'm currently working on a UI overhaul which includes showing a help for show's parameter when typing '`show `'. I just pushed what I got so far to the `ui-overhaul` branch. It's not done yet, but it should compile, if you want to have a look. (Or grab a binary from the [CI](https://github.com/MalteT/2a-emulator/actions/runs/156472317))

In regards to documentation: I would like to move away from the Benutzerhandbuch.pdf. It was created mainly for the thesis and using the README.md feels less "static".

Thank you for offering your help! Feel free to add a section to the README.md explaining the usage in more depth.
I'm using [cargo-readme](https://crates.io/crates/cargo-readme) to generate the README.md from the documentation at the start of [`src/main.rs`](https://github.com/MalteT/2a-emulator/blob/master/src/main.rs) to keep them in sync. Let me know if you have any trouble with that.

It would also be nice to have more options available to the `show` command, so let me know if you have any ideas about what might be interesting to have.

Regards,
Malte
